data/is_ocaml_yet/multicore.md

```diff
@@ -0,0 +1,26 @@
+---
+id: multicore
+question: Is OCaml multicore yet?
+answer: OCaml 5 is out, now we need to build an ecosystem!
+categories:
+  - name: Async IO
+    status: 🆘
+    description: |
+      n/a
+    packages: []
```

Comment:
```suggestion
    packages:
    - [eio](https://github.com/ocaml-multicore/eio)
```

Reply:
Thanks, I added a few packages to the Multicore page, including eio